src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
+    transform-origin: bottom center;
+    animation: spin-minute 3600s steps(60) infinite;
```

Comment:
The animation `spin-minute` is set to take 3600 seconds and complete in 60 steps. This means each step will last 60 seconds. Verify that this is the intended behavior for the minutes hand animation.